azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -612,24 +561,20 @@ def _check_processes_in_agent_cgroup(self):
             """
             unexpected = []
             agent_cgroup_proc_names = []
-            # Now we call _check_processes_in_agent_cgroup before we enable the cgroups or any one of the controller is not mounted, agent cgroup paths can be None.
-            # so we need to check both.
-            cgroup_path = self._agent_cpu_cgroup_path if self._agent_cpu_cgroup_path is not None else self._agent_memory_cgroup_path
-            if cgroup_path is None:
-                return
+
             try:
                 daemon = os.getppid()
                 extension_handler = os.getpid()
                 agent_commands = set()
                 agent_commands.update(shellutil.get_running_commands())
                 systemd_run_commands = set()
                 systemd_run_commands.update(self._cgroups_api.get_systemd_run_commands())
-                agent_cgroup = self._cgroups_api.get_processes_in_cgroup(cgroup_path)
+                agent_cgroup_proccesses = self._agent_cgroup.get_processes()
```

Comment:
Note, before we were getting processes in only CPU cgroup.procs (or only Memory cgroups.procs if cpu wasn't mounted). Now this gets processes under cpu and memory if both mounted.